src/plugins/irc-events/error.js

```diff
@@ -8,8 +8,14 @@ module.exports = function(irc, network) {
 	irc.on("irc error", function(data) {
 		let text = data.error;
 
-		if (data.reason) {
-			text = data.reason + " (" + text + ")";
+		if (data.channel) {
+			text = `${data.channel}: `;
+		}
+
+		if (data.error === "user_on_channel") {
+			text += `User (${data.nick}) is already on channel`;
+		} else {
+			text += (`${data.reason} (${data.error})` || data.error);
```

Comment:
This fallback doesn't make particular sense, did you want to do something like `data.reason ? `${data.reason} (${data.error})` : data.error`?

Reply:
Is it okay to replace it with `text += (data.reason ? `${data.reason} (${data.error})` : data.error);`?

Reply:
`else if` ends up cleaner here, imo.